webroot/src/router/router.spec.ts

```diff
@@ -0,0 +1,34 @@
+//
+//  router.spec.ts
+//  InspiringApps modules
+//
+//  Created by InspiringApps on 4/12/20.
+//
+
+import guards from '@router/_guards';
+import store from '@/store';
+
+const chaiMatchPattern = require('chai-match-pattern');
+const chai = require('chai').use(chaiMatchPattern);
+
+const { expect } = chai;
+
+describe('Router Guards', () => {
+    it('should successfully return authentication guard', () => {
+        const result = guards.authGuard();
+
+        expect(result).to.equal(true);
+    });
+    it('should successfully return no-authentication guard (default)', () => {
+        const result = guards.noAuthGuard();
+
+        expect(result).to.equal(true);
+    });
```

Comment:
How can both these cases pass? Wouldn't the `authGuard()` return false since no one is logged in?

Reply:
Good question. The guards themselves currently don't have much going on other than evaluating the store. The auth-guard route is set to return true specifically for the test runner, as a convenience to not have to setup login mocks for each test. If that's something you'd prefer we change, let me know. Right now it saves us a decent chunk of time in writing tests.

Reply:
Yep, that all sounds correct. I added a comment on that test.